### ISSUE TITLE
less optimization if image is already small

### ISSUE THREAD

**Issue by ineagu** [Codeinwp/optimole-wp#111]
@selul not sure if we can find some smart solutions for cases when the image is already too small or optimized.

For example, we have this image here: https://www.dropbox.com/s/iwgmi1hxqi5t5ig/Screenshot%202019-07-13%2018.02.17.png?dl=0, it seems to be optimized and is half of the size it needs for retina. Once I put it through optimole it gets bad in terms of quality: https://www.dropbox.com/s/1jsnll9qgz2hzi8/Screenshot%202019-07-13%2018.04.26.png?dl=0.

In other words, try to figure out if in some cases the perceived quality dropped too much, probably nothing to do here right now
